recipes/apache-airflow-providers-trino/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "apache-airflow-providers-trino" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/apache-airflow-providers-trino-{{ version }}.tar.gz
+  sha256: 9697c765a791cd868e5783d112332e3d46d88ccc61ae90ec937081b35697cd1e
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - wheel
```

Comment:
Is this actually required? I don't see it getting used in `setup.py`...

Reply:
Fair enough.  `grayskull` adds it based on the `pypi` metadata but that's presumably because a wheel gets built for pypi.